python/lsst/analysis/tools/actions/plot/scatterplotWithTwoHists.py

```diff
@@ -672,17 +672,17 @@ def _makeTopHistogram(
     ) -> None:
         # Top histogram
         topHist = figure.add_subplot(gs[0, :-1], sharex=ax)
+        x_min, x_max = ax.get_xlim()
+        bins = np.linspace(x_min, x_max, 100)
 
-        if "any" in self.plotTypes:
-            x_all = data[f"x{self._datatypes['any'].suffix_xy}"]
-            keys_notall = [x for x in self.plotTypes if x != "any"]
+        if "all" in self.plotTypes:
+            x_all = f"x{self._datatypes['all'].suffix_xy}"
```

Comment:
Mix of " vs '.

Reply:
It needs both to have the string key inside the string.